OrangeHRMTests/Tests/Tests.cs

```diff
@@ -0,0 +1,590 @@
+            GenericPages.PIMPage.ChoseScreen();
+            GenericPages.PIMPage.ChoseType();
+            GenericPages.PIMPage.ClickSaveButton();
+            GenericPages.BasePage.TopBarMenu.ClickTopbarMenuButtonByName(PIMPageTopBarMenuButtonsNames.EmployeeListTopBarMenuButtonName);
+
+            ClassicAssert.AreEqual(PIMPageHeadersTextValues.EmployeeInformationHeaderTextValue, GenericPages.PIMPage.GetImployeeInformationText());
+
+            Table.ClickPencilEditButton();
+
+            ClassicAssert.AreEqual(NewFieldName, GenericPages.PIMPage.GetAddedCustomFieldNameTextElement());
+
+            GenericPages.PIMPage.EnterValueInInputTextField(NewFieldName, NewFieldValue);
+            GenericPages.PIMPage.ClickSaveCustomFieldButton();
+
+            ClassicAssert.AreEqual(NewFieldName, GenericPages.PIMPage.GetAddedCustomFieldNameTextElement());
+
+            GenericPages.PIMPage.EnterValueInInputTextField(NewFieldName, Keys.Control + "a" + Keys.Delete);
+            GenericPages.PIMPage.ClickSaveCustomFieldButton();
+
+            ClassicAssert.AreEqual(NewFieldName, GenericPages.PIMPage.GetAddedCustomFieldNameTextElement());
+
+            GenericPages.BasePage.TopBarMenu.ClickTopbarMenuButtonByName(PIMPageTopBarMenuButtonsNames.ConfigurationTopBarMenuButtonName);
+            GenericPages.BasePage.TopBarMenu.ClickTopbarMenuButtonByName(PIMPageTopBarMenuButtonsNames.CustomFieldsTopBarMenuButtonName);
+            Table.ClickTrashButton();
+            GenericPages.PIMPage.ClickConfirmDeletionButton();
+        }
+
+        [Test]
+        public void U_ValidateAssignSkillToEmployeeProfileTest()
+        {
+            const string NewSkillValue = "SQL";
+
+            GenericPages.BasePage.LeftMenuNavigationPanel.GoToLeftMenuItem(LeftMenuNavigationPanelButtonsNames.PIMButtonName);
+
+            GenericPages.BasePage.TopBarMenu.ClickTopbarMenuButtonByName(PIMPageTopBarMenuButtonsNames.EmployeeListTopBarMenuButtonName);
+
+            ClassicAssert.AreEqual(PIMPageHeadersTextValues.EmployeeInformationHeaderTextValue, GenericPages.PIMPage.GetImployeeInformationText());
+
+            Table.ClickPencilEditButton();
+            GenericPages.PIMPage.ClickQualificationsButton();
+            GenericPages.PIMPage.ClickAddSkillsButton();
+            GenericPages.PIMPage.ClickDropDownListArrowButtonByName(DropDownFieldsNames.SkillDropDownFieldName);
+            MyWebElement.SelectValueFromOrangeDropdownList(NewSkillValue);
+            GenericPages.PIMPage.ClickSaveButton();
+
+            ClassicAssert.AreEqual(NewSkillValue, Table.GetValueOfTextFieldByName(TableColumnsNames.SkillTableColumnName));
+
+            GenericPages.PIMPage.ClickSkillsTableTrashButton();
+            GenericPages.PIMPage.ClickConfirmDeletionButton();
+
+            GenericPages.BasePage.TopBarMenu.ClickArrowButton();
+            GenericPages.BasePage.TopBarMenu.ClickUserDropdownItemByName(TopBarUserDropDownButtons.LogoutButton);
```

Comment:
it looks like your tests depend on the order you execute it, it shouldn't be like this
think how you can refactor it

Reply:
Fixed